craco.config.js

```diff
@@ -1,4 +1,6 @@
+// const { whenProd } = require('@craco/craco')
```

Comment:
remove this line pls

Reply:
Good catch. I will remove it with another PR to avoid a re-build of releases just for this small thing.